### DISCUSSION TITLE
Great mod, thank you

### DISCUSSION THREAD

**Discussion by Schlappspaten** [AMacro/dv-multiplayer#42] [Show and tell]
Hi, I just gave it a try with few friends. We were up to 4 people playing and all worked (within the known limits) very well. We started out at a fresh game (in realistic mode), ran some trains, did some shunting and after few hours without the slightest technical hick-up we rounded up the harbour. We were lucky with lots of 1 or 2 group shunting jobs ending in preparing three big trains, one for each:
https://www.youtube.com/watch?v=4j7S0UTBK3I

We all ran to our different locations, not without damaging a few cars and engines due to user error or multiplayer imposed distraction.

Guy(s), this is a great mod that works well although it has hidden dangers ;). Can't wait to see it developing further.

Thank you and keep up the good work!

**Comment by AMacro** [comment]
It makes us so happy to hear people are playing and enjoying this mod!
We love seeing the videos people are making.

Please let us know if you find any more bugs or issues; we'll get to them all eventually, but will triage them
